src/graphics/program-lib/chunks/specular.frag

```diff
@@ -6,6 +6,10 @@ uniform vec3 material_specular;
 uniform sampler2D texture_specularMap;
 #endif
 
+#ifdef CLEARCOAT
+uniform float material_clear_coat_specularity;
+#endif
+
 void getSpecularity() {
```

Comment:
The naming conventions in the code seem really inconsistent. addSomething should add to something, get/calcSomething should return it. As you touch the shader code, would be great if you could bring consistency where possible.

(Just to point out this function called getSpecularity is really adding specularity).

Reply:
hmm - I think it is getting the specularity in this case? A bigger rework of the material system (which we have discussed a bit before) is probably beyond the scope of this change?

Reply:
Oh I was referring to the fact the function doesn't actually return anything. It's called getSpecularity, but it's actually calculating (or perhaps setting?) specularity. Anyway, don't worry at all in the PR if it's a big change. I just thought it'd be nice if we decide on naming convention and then nudge the code to be more consistent as we go :)

Reply:
Ah - I get what you mean - the function is getting the value based on the input options (texture and/or uniform) and then setting the global variable - so it's really initializing the value - so maybe renaming to initSpecularity() makes more sense? (in a separate PR)